### PR TITLE
HOUSNAV-25: DT for Workflow 2 + Sysdig Team Setup

### PR DESCRIPTION
**Pull request by jbook24**
initial commit

[HOUSNAV-25](https://hous-bssb.atlassian.net/browse/HOUSNAV-25)

## Overview & Purpose
- These changes set the ground work for the initial Data Transformation for Workflow 2. 
- These changes also house initial team config for Sysdig logging. 

## Summary of Changes
- Initial transformation of 9.10.14 along with test 
- sysdig-team-cr.yaml file 

## Side Effects
**Update sysdig-team-cr.yaml** 
- Login to OpenShift Console via terminal 

**Make updates to the Sysdig Team Profile** 
- Run oc edit -f sysdig-team-cr.yaml  

**View saved changes**
- Run oc describe sysdig-team d6af69-sysdigteam   

## Notes & Resources
Sysdig Onboarding [Reference](https://developer.gov.bc.ca/docs/default/component/platform-developer-docs/docs/app-monitoring/sysdig-monitor-onboarding/)

### CODE REVIEW

File: apps/web/sysdig-team-cr.yaml [hunk 1]
```diff
@@ -0,0 +1,17 @@
+apiVersion: ops.gov.bc.ca/v1alpha1
+kind: SysdigTeam
+metadata:
+  name: d6af69-sysdigteam
+  namespace: d6af69-tools
+spec:
+  team:
+    description: The Sysdig Team for the OpenShift Project Set d6af69.
+    users:
+      - name: joshua.booker@gov.bc.ca
```
Comment (wardm89) [inline apps/web/sysdig-team-cr.yaml:10]: can we add Denys to this yet?

File: packages/data/walkthroughs/9.10.14.test.ts [hunk 2]
```diff
@@ -0,0 +1,46 @@
...
+          return false;
+        }
+
+        return !Object.values(data.sections).some((section) =>
+          section.sectionQuestions.includes(questionId),
+        );
+      },
+    );
+    if (possibleUnusedQuestion)
+      console.log("Possible unused question ID:", possibleUnusedQuestion[0]);
```
Comment (wardm89) [inline packages/data/walkthroughs/9.10.14.test.ts]: Do we need this if the test is checking if it is undefined?

File: packages/data/walkthroughs/9.10.14.json [hunk 3]
```diff
@@ -0,0 +1,738 @@
+{
+  "info": {
+    "title": "Test Case 2 - Spatial Separation Between Buildings",
+    "startingQuestionId": "intro"
+  },
+  "sections": {
+    "intro": {
+      "sectionTitle": "TODO - Introduction",
```
Comment (wardm89) [inline packages/data/walkthroughs/9.10.14.json]: Who will do this?

File: apps/web/sysdig-team-cr.yaml (hunk 1 above)
Comment (jbook24) [reply apps/web/sysdig-team-cr.yaml:10]: Not yet, awaiting account setup

**Comment by nicholaslyon-dt** [comment]
I've finished my updates to the JSON for the walkthrough. I did not have time to update all the TypeScript types. I ended up having to add more things than I thought.

**Pull request merged**